test/stablex/large_exchange.js

```diff
@@ -0,0 +1,70 @@
+const BatchExchange = artifacts.require("BatchExchange")
+const MockContract = artifacts.require("MockContract")
+const IdToAddressBiMap = artifacts.require("IdToAddressBiMap")
+const IterableAppendOnlySet = artifacts.require("IterableAppendOnlySet")
+
+const { closeAuction } = require("../../scripts/stablex/utilities.js")
+
+const { longRingTrade, solutionSubmissionParams } = require("../resources/examples")
+const { makeDeposits, placeOrders, setupGenericStableX } = require("./stablex_utils")
+
+contract("BatchExchange", async accounts => {
```

Comment:
Why it's own file?

Reply:
No particular reason. But it isn't a unit test if that makes a difference.

Reply:
Moved it into the test file and included assertions.